### PR TITLE
Update QUEST_en.tsv

### PR DESCRIPTION
**Pull request by megaoka**
It's getting harder and harder to find changes to make!

### CODE REVIEW

File: QUEST_en.tsv [hunk 1]
```diff
@@ -26,9 +26,9 @@ QUEST_20150317_000025	Evil forces and will be but I do not feel at all. {Nl} The
 QUEST_20150317_000026	Okay, Please go back to Uska.{nl}He will tell you where to go.{nl}May the Goddess bless for your future..
 QUEST_20150317_000027	Please select preferred Smart Gen.
 QUEST_20150317_000028	William
-QUEST_20150317_000029	It will be 100 Silvers.{nl}This is cheaper compared to going straight into a bunch of monsters.
+QUEST_20150317_000029	It will be 100 Silvers.{nl}This is cheaper compared to going through a bunch of monsters.
```
Comment (emailboxu) [inline QUEST_en.tsv]: compared to -> than

File: QUEST_en.tsv [hunk 2]
```diff
@@ -26,9 +26,9 @@ QUEST_20150317_000025	Evil forces and will be but I do not feel at all. {Nl} The
...
 QUEST_20150317_000030	
-QUEST_20150317_000031	It will be 100 Silvers.{nl}This amount includes operating, labor and other expenses.
+QUEST_20150317_000031	It will be 100 Silvers.{nl}This includes operating, labor and other expenses.
```
Comment (emailboxu) [inline QUEST_en.tsv:31]: Personally am a fan of the Oxford comma (after labor), optional though.

Comment (megaoka) [inline QUEST_en.tsv:31]: I've never really used it. Personal habit I guess. :P

**Comment by Camarilla01** [comment]
Can we translate to other langueses, like spanish?

**Comment by megaoka** [comment]
Probably, I bet q project could be made if enough people were interested. I would submit it in the Issues tab.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**